Chemical-starting-points.md

```diff
@@ -0,0 +1,45 @@
+2. Provide an OpenMM `Topology` which includes bond orders and thus can be converted to molecules corresponding to the components of your system
+
+Without this information, identifying the chemical identity of the molecules comprising your system becomes a matter of guesswork rather than certainty.
+Different molecules can have the same chemical graph but differ in bond order and formal charge, or different resonance structures may be treated rather differently by some force fields (e.g. `c1cc(ccc1c2cc[nH+]cc2)[O-]` vs `C1=CC(C=CC1=C2C=CNC=C2)=O`, where the central bond is rotatable in one resonance structure but not in the other) even though they have identical formal charge and connectivity (chemical graph).
+A force field which uses the chemical identity of molecules to assign parameters needs to know what those molecules are, first.
+
+## Can I use an AMBER (or GROMACS) topology/coordinate file as a starting point for applying a SMIRNOFF force field?
+
+In a word, "no".
+
+Parameter files used by typical molecular dynamics simulation packages do not currently encode enough information to identify the molecules chemically present, or at least not without drawing inferences.
+For example, one could take a structure file and infer bond orders based on bond lengths, or attempt to infer bond orders from force constants in a parameter file.
+Such inference work is outside the scope of SMIRNOFF.
+
+If you have such an inference problem, we recommend that you use pre-existing cheminformatics tools available elsewhere (such as via the OpenEye toolkits, such as the `OEPerceiveBondOrders` functionality offered there) to solve this problem and identify your molecules before beginning your work with SMIRNOFF.
+
+## What about starting from a PDB file?
+
+PDB files do not in general provide the chemical identity of small molecules contained therein, and thus do not provide suitable starting points for applying SMIROFF to small molecules.
+This is especially problematic for PDB files from X-ray crystallography which typically do not include proteins, making the problem even worse.
+For our purposes here, however, we assume you begin with the coordinates of all atoms present and the full topology of your system.
+
+Given a PDB file of a hypothetical biomolecular system of interest containing a small molecule, there are several routes available to you for treating the small molecule present:
+- Use a cheminformatics toolkit (see above) to infer bond orders
+- Identify your ligand from a database; e.g. if it is in the Protein Data Bank (PDB), it will be present in the [Ligand Expo](http://ligand-expo.rcsb.org) meaning that it has a database entry and code you can use to look up its putative chemical identity
+- Identify your ligand from the literature or your collaborators: If the above fail, you will have to find out what ligand is supposed to be in the structure you are working with.
```

Comment:
The colon after collaborators here seems off. shouldn't the "If the above... " sentence be on a new line outside the list?

Reply:
No, the last bullet is intended to be a single thought -- if you can't use a database or a toolkit, you have to work with someone/the literature to figure out what molecule they meant. Maybe I should rephrase a bit, suggestions?

Reply:
Oh, that makes more sense. It seemed obvious that the third point is an alternative to the other two. Also all three are options for finding out what ligand is supposed to be used so maybe... 

"Identify your ligand by name or SMILES string from the literature or your collaborators"